pkgs/applications/graphics/apitrace/default.nix

```diff
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   name = "apitrace-${version}";
-  version = "7.1";
+  version = "7.1-git-20170411";
```

Comment:
@bjornfor: `git describe` isn't bullet-proof in general, unfortunately; for example: `7.1-363-gSomeHash` < `7.1-72-gSomeHash`.